apps/design/backend/migrations/1756159958924_organizations-table.js

```diff
@@ -33,6 +33,14 @@ exports.up = async (pgm) => {
       )`
       );
     }
+  } else {
+    // In dev mode, create the default dev organization (use VotingWorks org ID to get full features)
+    pgm.sql(`
+      INSERT INTO organizations (id, name) VALUES (
+        'votingworks',
```

Comment:
You should be able to import this as `votingWorksOrgId()` from `../build/globals' (to ensure it matches)